packages/ra-core/src/util/LabelPrefixContextProvider.tsx

```diff
@@ -1,17 +1,9 @@
 import * as React from 'react';
 import { LabelPrefixContext } from './LabelPrefixContext';
-import { useLabelPrefix } from './useLabelPrefix';
 
-export const LabelPrefixContextProvider = ({
-    prefix,
-    concatenate = true,
```

Comment:
Looks good but I wonder why we didn't set this to `false` on the `Form`

Reply:
Yes we could 😅. But `LabelPrefixContext` is not used elsewhere, so there is no point in keeping this logic IMO.